### PR TITLE
Add Microsoft License to the top of all source files in Learn-LTI

### PR DESCRIPTION
**Pull request by adbindal**
## Description
Per legal guidance, adding the following header to all the source files in Learn-LTI
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
```

> Added a few more changes like updating `manifest.json` content and removing unused `ServiceWorker` code

### CODE REVIEW

File: backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Properties/launchSettings.json:5]: lets add an empty line here

File: backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/Properties/launchSettings.json:5]: lets add an empty line here also

File: backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/host.json [hunk 3]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/host.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Assignments/Edna.Assignments/host.json [hunk 4]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Assignments/Edna.Assignments/host.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Connect/Properties/launchSettings.json [hunk 5]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Connect/Properties/launchSettings.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Connect/host.json [hunk 6]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Connect/host.json:5]: lets add an empty line here also

File: client/tsconfig.json [hunk 7]
```diff
@@ -1,3 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
 {
```
Comment (nehalilani) [inline client/tsconfig.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Users/Edna.Users/host.json [hunk 8]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Users/Edna.Users/host.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Users/Edna.Users/Properties/launchSettings.json [hunk 9]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Users/Edna.Users/Properties/launchSettings.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Platforms/Edna.Platforms/host.json [hunk 10]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Platforms/Edna.Platforms/host.json:5]: lets add an empty line here also

File: backend/Functions/Edna.Platforms/Edna.Platforms/Properties/launchSettings.json [hunk 11]
```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {
```
Comment (nehalilani) [inline backend/Functions/Edna.Platforms/Edna.Platforms/Properties/launchSettings.json:5]: lets add an empty line here also

**Review by nehalilani — COMMENTED** [review]
Just want to clarify a few things that were coming to my mind- 
1. Is there a specific reason why we haven't left an extra empty line in launchSettings.json and host.json?
2. The header is different for client side files and backend files. Shouldn't they be the same?
3. We should add the header to the deployment files as well

File: backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Properties/launchSettings.json (hunk 1 above)
Comment (adbindal) [reply backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Properties/launchSettings.json:5]: An empty line was not increasing any readability in this file, instead it was seeming a bit more like noise. Which is why I left it as is.

File: backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/Properties/launchSettings.json (hunk 2 above)
Comment (adbindal) [reply backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/Properties/launchSettings.json:5]: Same as above.

**Comment by adbindal** [comment]
> Just want to clarify a few things that were coming to my mind-
> 
> 1. Is there a specific reason why we haven't left an extra empty line in launchSettings.json and host.json?
> 2. The header is different for client side files and backend files. Shouldn't they be the same?
> 3. We should add the header to the deployment files as well

1. I was focused on the readability of the file and none of the `.json` files were feeling more readable with an empty line. I don't have strong opinions though.
2. I agree. Unfortunately, that seems to be how it has been inside some of the other microsoft's github projects I looked at. Based on my shallow investigation, I copied header from `.ts` files and `.cs` files. I'll update the license to be consistent if that seems to be the right thing according to you.
3. None of the other microsoft's github projects have placed a license header in powershell or ARM templates. It seems like they are not source files, but script files. Which is why I also left them as is.

**Review by nehalilani — APPROVED** [review]
Looks good to me

**Pull request merged**